File: src/File.Application/DTO/Response/Folder/FolderResponseDto.cs
namespace File.Application.DTO.Response.Folder;

public class FolderResponseDto
{
    public string? Name { get; set; }
    public string? Path { get; set; }
    public short Size { get; set; }
    public DateTime CreateDate { get; set; }
    public int UserId { get; set; }
    public bool IsDeleted { get; set; }
    public DateTime? DeletedDate { get; set; }
}

File: src/File.Api/ClientApp/src/app/models/Folder/folderResponseDto.ts
export interface folderResponseDto {
    success: boolean;
    message: string;
    data: Data;
}

interface Data {
    id: number;
    name: string;
    path: string;
    size: number;
    createDate: string;
    userId: number;
    isDeleted: boolean;
    deletedDate: any;
    files: any[];
    user: string;
}
